docs/topics/con_the-externalized-configuration-design-pattern.adoc

```diff
@@ -7,7 +7,7 @@ This allows the application configuration to change as it moves through differen
 Externalizing the configuration also keeps sensitive or internal information out of your code base and version control.
 Many languages and application servers provide environment variables to support externalizing an application's configuration.
 
-Microservices architectures and polyglot environments add a layer of complexity to managing an application's configuration.
+Microservices architectures and multi-language environments add a layer of complexity to managing an application's configuration.
```

Comment:
I actually think this is a change for the worse. "Polyglot programming" is a well established technical term.

Reply:
It is, but it expects knowledge of programming jargon (verified that with a few colleagues—roughly half of them hasn't heard it before and had to infer its meaning). As RHOAR is suited also for people with only basic skills, we can't afford to make that assumption. I can put _polyglot_ into parentheses as an explanation, but it can't stay there as the sole carrier of that information.

Reply:
I confess I live in a bubble :-) "Multi-language" is fine too.